src/components/Activity/DefaultTabBar.js

```diff
@@ -0,0 +1,125 @@
+const React = require('react');
+const ReactNative = require('react-native');
+const {
+  Dimensions,
+  StyleSheet,
+  Text,
+  View,
+  Animated,
+} = ReactNative;
+const Button = require('./Button');
+
+const DefaultTabBar = React.createClass({
+  propTypes: {
+    goToPage: React.PropTypes.func,
+    activeTab: React.PropTypes.number,
+    containerWidth: React.PropTypes.number,
+    leftOffset: React.PropTypes.number,
+    rightOffset: React.PropTypes.number,
+    tabs: React.PropTypes.array,
+    backgroundColor: React.PropTypes.string,
+    activeTextColor: React.PropTypes.string,
+    inactiveTextColor: React.PropTypes.string,
+    textStyle: Text.propTypes.style,
+    tabStyle: View.propTypes.style,
+    style: View.propTypes.style,
+    renderTab: React.PropTypes.func,
+    underlineStyle: View.propTypes.style,
+  },
+
+  getDefaultProps() {
+    return {
+      activeTextColor: 'navy',
+      inactiveTextColor: 'black',
+      backgroundColor: null,
+    };
+  },
+
+  renderTab(name, page, isTabActive, onPressHandler) {
+    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const textColor = isTabActive ? activeTextColor : inactiveTextColor;
+    const fontWeight = isTabActive ? 'bold' : 'normal';
+
+    return <Button
+      style={{flex: 1, }}
+      key={name}
+      accessible={true}
+      accessibilityLabel={name}
+      accessibilityTraits='button'
+      onPress={() => onPressHandler(page)}
+    >
+      <View style={[styles.tab, this.props.tabStyle, ]}>
+        <Text style={[{color: textColor, fontWeight, }, textStyle, ]}>
```

Comment:
23: A space is required after '{'
52: Unexpected trailing comma
66: Unexpected trailing comma
68: There should be no space before ']